config/watcher/gateway.yaml

```diff
@@ -20,5 +20,5 @@ spec:
         number: 443
         protocol: HTTPS
       tls:
-        credentialName: klm-watcher
+        credentialName: istio-gateway-secret
```

Comment:
Can you shortly explain why we have to change the credentailName here? also in this PR, you name it [gateway-secret](https://github.com/kyma-project/lifecycle-manager/pull/1871/files#diff-076f5cb5879121781dc0b5aaf99c461555cbab02b46a9b13b94482242f0aa985R23), and we previously have issue regarding alignment of resource name, https://github.com/kyma-project/lifecycle-manager/pull/1721#issuecomment-2256185803, please also have a reference to that decision for this renaming.

Reply:
If it make sense to have a short meeting to sync the decision, please don't hesitate bring it to team, you can sync this with @Tomasz-Smelcerz-SAP first.

Reply:
would `klm-istio-gateway` be more fitting then?

Reply:
I would vote for `klm-istio-gateway`

Reply:
I'll use that for now, but let's discuss it briefly after a daily.